### DISCUSSION TITLE
How to run the engine without NNUE?

### DISCUSSION THREAD

**Discussion by nguyenpham** [official-pikafish/Pikafish#23] [General]
I want to test the engine with and without NNUE but can't turn NNUE off. It doesn't have any option such as "Use_NNUE". I have tried to set option EvalFile to empty by commands ```setoption name EvalFile value <empty>``` or ```setoption name EvalFile value``` but they didn't work.

**Comment by PikaCat-OuO** [comment]
See https://github.com/official-pikafish/Pikafish/issues/11, and a newer net will be released soon. You can always find the latest net at https://github.com/official-pikafish/Pikafish/releases.
